pxtlib/sprite-editor/canvasGrid.ts

```diff
@@ -104,10 +110,7 @@ namespace pxtsprite {
                 context.fillStyle = this.palette[color - 1];
                 context.fillRect(x, y, this.cellWidth, this.cellHeight);
             }
-            else if (transparency) {
-                context.clearRect(x, y, this.cellWidth, this.cellHeight);
```

Comment:
Why is this removed?

Reply:
Since the floating layer is drawn onto the same layer as as the paint layer, if it has transparency, this code will remove the color of the painted layer below it. Changed it so that it clears the whole layer before painting anything instead of clearing on a pixel by pixel basis.